nixos/modules/services/misc/paperless.nix

```diff
@@ -134,6 +134,17 @@ in
       description = lib.mdDoc "Whether all users can write to the consumption dir.";
     };
 
+    createDirs = mkOption {
```

Comment:
I would recommend instead to use mkForce for the specific paths in tmpfilesd as this option is very specific to your exact setup and not generally useful IMO.

Reply:
I guess what @SuperSandro2000 means is to convert `tmpfiles.rules` to `tmpfiles.settings`, like so:

```nix
systemd.tmpfiles.settings."10-paperless" = let
  defaultRule = {
    inherit (cfg) user;
    inherit (config.users.users.${cfg.user}) group;
  };
in {
  "${cfg.dataDir}".d = defaultRule;
  "${cfg.mediaDir}".d = defaultRule;
  "${cfg.consumptionDir}".d = if cfg.consumptionDirIsPublic then { mode = "777"; } else defaultRule;
};
```

Then you can use the module system's builtin features to disable dir creation:
```nix
systemd.tmpfiles.settings."10-paperless" = mkForce {};
```

Yes, this approach is better than an extra option!

Reply:
great, thanks for the feedback. I'll refactor the PR

Reply:
Refactored, thanks for the feedback.